pkgs/development/interpreters/php/default.nix

```diff
@@ -223,35 +226,19 @@ let
     };
 
 in {
-  # Because of an upstream bug: https://bugs.php.net/bug.php?id=76826
-  # We can't update the darwin versions because they simply don't compile at
-  # all due to a bug in the intl extensions.
-  #
-  # The bug so far is present in 7.1.21, 7.1.22, 7.1.23, 7.2.9, 7.2.10, 7.2.11.
+  php71 = generic {
+    version = "7.1.23";
+    sha256 = "0jyc5q666xh808sgy78cfylkhy5ma2zdg88jlxhagyphv23aly9d";
+
+    # https://bugs.php.net/bug.php?id=76826
+    extraPatches = optional stdenv.isDarwin [ ./php71-darwin-isfinite.patch ];
```

Comment:
The `[]` brackets are useless here, they're already added by `optional`, so `patches` ends up containing `[ ./fix-paths-php7.patch [ ./php71-darwin-isfinite.patch ] ]`, which I guess nix handles correctly, but… :)